docs/v1.md

```diff
@@ -263,67 +238,84 @@
 - uniqueItems : `boolean` | `undefined`
 - writeOnly : `boolean` | `undefined`
 
+## Schemas
+- all() : `Schema[]`
+- get(id: `string`) : `Schema` | `undefined`
+- has(id: `string`) : `boolean`
+- isEmpty() : `boolean`
+
 ## SecurityScheme
+- id() : `string`
 - bearerFormat() : `string` | `undefined`
 - description() : `string`
-- extension(name) : `any` | `undefined`
-  - name : `string`
+- extensions() : `Extensions`
 - flows() : `OAuthFlows`
 - hasBearerFormat() : `boolean`
 - hasDescription() : `boolean`
-- hasExtension(name) : `boolean`
-  - name : `string`
 - in() : `enum{'user', 'password', 'query', 'header', 'cookie'}`
 - openIdConnectUrl() : `string`
 - scheme() : `string` | `undefined`
 - scopes() : `string[]`
 - type() : `enum{'userPassword', 'apiKey', 'X509', 'symmetricEncryption', 'asymmetricEncryption', 'httpApiKey', 'http', 'oauth2', 'openIdConnect'}`
 
-## ServerVariable
-- allowedValues() : `any[]`
-- defaultValue() : `string` | `undefined`
-- description() : `string` | `undefined`
-- examples() : `string[]`
-- extension(name) : `any` | `undefined`
-  - name : `string`
-- hasAllowedValues() : `boolean`
-- hasDefaultValue() : `boolean`
-- hasDescription() : `boolean`
-- hasExtension(name) : `boolean`
-  - name : `string`
-- hasName() : `boolean`
-- name() : `string` | `undefined`
+# SecuritySchemes
+- all() : `SecurityScheme[]`
+- get(id: `string`) : `SecurityScheme` | `undefined`
+- has(id: `string`) : `boolean`
+- isEmpty() : `boolean`
+
+## Servers
+- all() : `Server[]`
+- filterBy(filter : `(server: Server) => boolean`) : `Server[]`
+- filterBySend() : `Server[]`
+- filterByReceive() : `Server[]`
+- get(id: `string`) : `Server` | `undefined`
+- has(id: `string`) : `boolean`
```

Comment:
```suggestion
- get(name: `string`) : `Server` | `undefined`
- has(name: `string`) : `boolean`
```

Maybe the parameter should be `name` since the `server` object does not have `id` parameter.

Reply:
The fact that Server had a name field was wrong. It should be id since servers doesnt have such a name so the Id becomes a match with the key of the map on the spec `servers` section.

I fixed it! Thanks for the catch.